src/main/java/core/basesyntax/figures/Square.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.figures;
+
+public class Square extends Figure {
+    private int side;
+
+    public Square(String color, String name, int side) {
+        super(color, name = "Square");
```

Comment:
In the constructor, the assignment `name = "Square"` is unnecessary since `name` is being set through the `super` call. You should remove `name = "Square"` and just call `super(color, "Square")`.